### PR TITLE
[TPU Provisioner] Remove old diagram; use static cluster version

### PR DESCRIPTION
**Pull request by danielvegamyhre**
- Remove old diagram which uses a Job instead of JobSet
- Update instructions to use static cluster version

### CODE REVIEW

File: tpu-provisioner/README.md [hunk 1]
```diff
@@ -8,20 +8,22 @@ The provisioning process starts with an unschedulable "leader" pod (pod with Job
...
-Node Pools are cleaned up when the JobSet whose pods triggered the node pool creation is either completed, failed, or deleted.
+Node Pools are cleaned up when the JobSet whose pods triggered the node pool creation is either **completed, failed, or deleted**.
 
 ## Setup
 
-### Create a GKE Cluster with workload identity enabled
+### Create a GKE Cluster with workload identity enabled and a static cluster version
 
-The TPU Provisioner requires workload identity to be enabled. 
+The TPU Provisioner requires workload identity to be enabled, and cannot be on a release channel (auto upgrades
```
Comment (jedwins1998) [inline tpu-provisioner/README.md:17]: Can we change `workload identity` to `Workload Identity Federation for GKE`?

The seems to be the official name from reading the docs at https://cloud.google.com/kubernetes-engine/docs/concepts/workload-identity.

Comment (danielvegamyhre) [reply tpu-provisioner/README.md]: Sure, updated.

File: tpu-provisioner/README.md [hunk 2]
```diff
@@ -8,20 +8,24 @@ The provisioning process starts with an unschedulable "leader" pod (pod with Job
...
+Node Pools are cleaned up when the JobSet whose pods triggered the node pool creation is either **completed, failed, or deleted**.
 
 ## Setup
 
-### Create a GKE Cluster with workload identity enabled
+### Create a GKE Cluster with workload identity enabled and no release channel
 
-The TPU Provisioner requires workload identity to be enabled. 
+The TPU Provisioner requires [Workload Identity Federation for GKE](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) to be enabled, and cannot be on a release channel (auto upgrades
+are disabled on node pools created by the TPU provisioner, to minimize disruptions to training workloads)
```
Comment (jedwins1998) [inline tpu-provisioner/README.md:18]: There is not a period at the end of the sentence "The TPU Provisioner requires...".

**Review by jedwins1998 — APPROVED** [review]

File: tpu-provisioner/README.md (hunk 2 above)
Comment (danielvegamyhre) [reply tpu-provisioner/README.md]: Woops, fixed.

File: tpu-provisioner/README.md [hunk 3]
```diff
@@ -8,20 +8,24 @@ The provisioning process starts with an unschedulable "leader" pod (pod with Job
...
-Node Pools are cleaned up when the JobSet whose pods triggered the node pool creation is either completed, failed, or deleted.
+Node Pools are cleaned up when the JobSet whose pods triggered the node pool creation is either **completed, failed, or deleted**.
 
 ## Setup
 
-### Create a GKE Cluster with workload identity enabled
+### Create a GKE Cluster with workload identity enabled and no release channel
 
-The TPU Provisioner requires workload identity to be enabled. 
+The TPU Provisioner requires [Workload Identity Federation for GKE](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) to be enabled, and cannot be on a release channel (auto upgrades
```
Comment (nstogner) [inline tpu-provisioner/README.md:17]: remove "Federation" - this is cross cloud WI

**Review by nstogner — APPROVED** [review]

File: tpu-provisioner/README.md (hunk 3 above)
Comment (jedwins1998) [reply tpu-provisioner/README.md:17]: Can you explain the difference? I used the Workload Identity Federation for GKE docs when setting up the TPU Provisioner. How is Workload Identity different?

Comment (danielvegamyhre) [reply tpu-provisioner/README.md]: Removed

**Pull request merged**